apps/minifront/src/state/dutch-auction/index.ts

```diff
@@ -118,13 +119,22 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
 
   auctionInfos: [],
   loadAuctionInfos: async (queryLatestState = false) => {
+    get().dutchAuction.loadAuctionInfosAbortController?.abort();
+    const newAbortController = new AbortController();
+
     set(state => {
       state.dutchAuction.auctionInfos = [];
+      state.dutchAuction.loadAuctionInfosAbortController = newAbortController;
     });
 
     /** @todo: Sort by... something? */
-    for await (const response of viewClient.auctions({ queryLatestState, includeInactive: true })) {
+    for await (const response of viewClient.auctions(
+      { queryLatestState, includeInactive: true },
+      { signal: newAbortController.signal },
```

Comment:
Weirdly, just passing the `newAbortController.signal` here didn't seem to have any effect, despite the [ConnectRPC docs](https://connectrpc.com/docs/web/cancellation-and-timeouts/) saying that it should work. I still left this line in, though, since it seems right and perhaps will be fixed in a later ConnectRPC release. But in the meantime, stopping the `for` loop seems to fix this issue.

Reply:
Can you add a code comment about this? Surprised it doesn't work.

Reply:
Yep, done